patches/chrome-browser-ui-browser.cc.patch

```diff
@@ -20,3 +20,11 @@ index 56dd79709e5b9cb13b5b48ccd312c3c97710eb6c..b96687e669758dff1915f0513cad9120
            tab_strip_model_delegate_.get(),
            params.profile,
            // Tab groups are disabled for app browsers.
+@@ -1336,6 +1336,7 @@ void Browser::OnWindowClosing() {
+   if (tab_restore_service && notify_restore_service) {
+     tab_restore_service->BrowserClosing(GetFeatures().live_tab_context());
+   }
++  BRAVE_BROWSER_ON_WINDOW_CLOSING
```

Comment:
TODO - just leaving a tracking comment here so I don't lose this when it gets rebased